### PR TITLE
#26 implemented class diagram entities

### PR DESCRIPTION
**Pull request by OussamaCherifi**
Implemented class diagram with existing relations.

### CODE REVIEW

File: quolance-api/src/main/java/com/quolance/quolance_api/entities/AssociatedTags.java
Comment (abdelh17) [inline quolance-api/src/main/java/com/quolance/quolance_api/entities/AssociatedTags.java:1]: remove this file for now

File: quolance-api/src/main/java/com/quolance/quolance_api/entities/Project.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    private String description;
+
+    private String status;
+
+    @ManyToMany(mappedBy = "projects", cascade = CascadeType.ALL)
+    private List<Application> applications;
+
+    @ManyToMany(mappedBy = "projects", cascade = CascadeType.ALL)
+    private List<AssociatedTags> associatedTags;
```
Comment (abdelh17) [inline quolance-api/src/main/java/com/quolance/quolance_api/entities/Project.java:26]: should be `List<Tag>` not `List<AssociatedTags>`

File: quolance-api/src/main/java/com/quolance/quolance_api/entities/Project.java
Comment (abdelh17) [inline quolance-api/src/main/java/com/quolance/quolance_api/entities/Project.java:1]: maybe missing link to client( user class), freelancer (user type)

**Review by abdelh17 — CHANGES_REQUESTED** [review]
Save the files that are to be removed on the side, they will be needed later in the project, but not in the near future. Other than that, there are just a few fields missing in a few classes to represent the relationships between classes.

**Review by abdelh17 — APPROVED** [review]

**Review by abdelh17 — CHANGES_REQUESTED** [review]
Set spring.jpa.hibernate.ddl-auto in application.properties to an environment variable to avoid messing up other DBs when pushing a new value

**Review by KA-devl — APPROVED** [review]

**Comment by KA-devl** [comment]
lgtm

**Review by abdelh17 — APPROVED** [review]

**Pull request merged**